packaging/pkg_helpers.bash

```diff
@@ -36,9 +36,7 @@ setup_cuda() {
   # Wheel builds need suffixes (but not if they're on OS X, which never has suffix)
   if [[ "$BUILD_TYPE" == "wheel" ]] && [[ "$(uname)" != Darwin ]]; then
     # The default CUDA has no suffix
```

Comment:
Should we remove this comment as well since all CUDA versions have a suffix now?